src/App.tsx

```diff
@@ -1,61 +1,122 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
```

Comment:
```suggestion
function getUserByID(userId: number): User | null {
```
It's better to specify. 
Btw, everywhere else you create arrow functions instead of function declaration, so it's better to do it here too.